.travis.yml

```diff
@@ -8,7 +8,7 @@ cache: pip
 matrix:
   include:
     - os: linux
-      python: 3.7
+      python: 3.6
```

Comment:
At some point we can revisit moving the doc deployment from travis -> circleCI.

Reply:
Yes, I was thinking of returning to that soon as well.